docs/source/data_extraction.rst

```diff
@@ -162,44 +162,40 @@ After iteratively adjusting the TableConfig with the ``preview`` method above, t
     table_config = table_configs.register(table_config)
     print("Definition registered as {}".format(table_config.definition_uid))
 
-Registered Table Configs can be built into GEM Tables.
-GEM Tables are sometimes large and time-consuming to build, so the build process is asynchronous.
-The steps are:
+Registered Table Configs can be built into GEM Tables. For example:
 
-1. Submit a GEM Table Build Job
-2. Poll the Job Status until it is a ``Success`` or ``Failure``
-3. (If success) Get the id and version for the table
-4. (If success) Download the table
+.. code-block:: python
 
+   table = project.tables.build_from_config(table_config)
+   project.tables.read(table, "./my_table.csv")
+   
+The above will build a table, wait for the build job to complete, and save the table locally.
+   
+However, GEM Tables are sometimes large and time-consuming to build, so the build process can be performed asynchronously with the ``initiate_build`` method.
 For example:
 
 .. code-block:: python
 
-    from time import sleep
-    # 1. Submit the GEM Table build job
-    job = table_configs.build_ara_table(table_config)
-    # 2. Poll the Job Status every second
-    while True:
-        status = project.table_configs.get_job_status(job.job_id)
-        if status.status in ['Success', 'Failure']:
-            break
-        sleep(1)
-    if status.status == 'Success':
-        # 3. Get the id and version for the table
-        table_id = status.output['display_table_id']
-        table_version = status.output['display_table_version']
-        # 4. Download the table
-        table = project.tables.get(table_id, table_version)
-        project.tables.read(table, "./my_table.csv")
-
-The return type of the ``build_ara_table`` method is a :class:`~citrine.resources.ara_job.JobSubmissionResponse` that contains a unique identifier for the submitted job.
+    job = project.tables.initiate_build(table_config)
+
+The return type of the ``initiate_build`` method is a :class:`~citrine.resources.job.JobSubmissionResponse` that contains a unique identifier for the submitted job.
 
 This identifier can be used to get the status of the job via the ``get_job_status`` method, which returns a :class:`~citrine.resources.ara_job.JobStatusResponse`.
 The :class:`~citrine.resources.ara_job.JobStatusResponse` contains a ``status`` string describing the state of the job and an ``output`` map that contains the table id and version.
```

Comment:
```suggestion
```
`get_job_status` is deprecated, as is the entire job status interface.  Short of a feature request to actually allow for user polling of the server, this should be removed from the documentation.  In digging up what would need to get done, at this point we'd either need to craft new methods or have the user roll their own URLs.